requirements_m1.txt

```diff
@@ -0,0 +1,48 @@
+absl-py==1.2.0
+astunparse==1.6.3
+cachetools==5.2.0
+certifi==2022.9.14
+charset-normalizer==2.1.1
+flatbuffers==2.0.7
+ftfy==6.1.1
+gast==0.4.0
+google-auth==2.11.0
+google-auth-oauthlib==0.4.6
+google-pasta==0.2.0
+grpcio==1.49.0
+h5py==3.7.0
+idna==3.4
+keras==2.10.0
+Keras-Preprocessing==1.1.2
+libclang==14.0.6
+Markdown==3.4.1
+MarkupSafe==2.1.1
+numpy==1.23.3
+oauthlib==3.2.1
+opt-einsum==3.3.0
+packaging==21.3
+Pillow==9.2.0
+protobuf==3.20.1
```

Comment:
Should be 3.19. Also, most of these dependencies here are transitive, the `requirements` should focus on required dependencies and let pip handle the rest.

Reply:
Done, thanks